Wrappers/Python/cil/io/ZEISSDataReader.py

```diff
@@ -22,14 +22,11 @@
 from cil.framework.labels import AngleUnit, AcquisitionDimension, ImageDimension
 import numpy as np
 import os
-import olefile
 import logging
+
 dxchange_logger = logging.getLogger('dxchange')
 dxchange_logger.setLevel(logging.ERROR)
```

Comment:
```suggestion
```

Reply:
Technically should [use a `contextmanager`](https://docs.python.org/3/howto/logging-cookbook.html#using-a-context-manager-for-selective-logging) to avoid unexpected side-effects, but really that's for  separate PR